learntools/__init__.py

```diff
@@ -1,3 +1,3 @@
-from . import advanced_pandas, core, deep_learning, gans, machine_learning, python, embeddings, ml_explainability, ml_insights, time_series_with_siraj
+from . import advanced_pandas, core, deep_learning, gans, machine_learning, python, embeddings, ml_explainability, ml_insights, time_series_with_siraj, data_viz_easy
 
-__version__ = '0.2.10'
```

Comment:
fyi: I bump the version frequently. This helps with debugging because we always know what code is in use.

We see version numbers even in the data about user code stored in the database (so we know what version of learntools evaluated each piece of their work).